lib/hanami/model/configurator.rb

```diff
@@ -25,12 +25,22 @@ class Configurator
       # @api private
       attr_reader :_schema
 
+      # @since x.x.x
+      # @api private
+      attr_reader :callbacks
+
       # @since 0.7.0
       # @api private
       def self.build(&block)
         new.tap { |config| config.instance_eval(&block) }
       end
 
+      # @since x.x.x
+      # @api private
+      def initialize
+        @callbacks = Hash[before: [], after: []]
```

Comment:
@pascalbetz @mereghost Yes, I agree with you. But we aim for actionable feedbacks. :wink:  Share your crazy names for this stuff! :joy_cat: